intro/journaling_api.md

```diff
@@ -2,39 +2,57 @@
+  - [Event expiry](#event-expiry)
+    - [Oldest available events](#oldest-available-events)
+    - [Fetching events from expired positions](#fetching-events-from-expired-positions)
+    - [Positon Validation](#positon-validation)
+    - [No Events in Journal](#no-events-in-journal)
 
 For enterprise developers, Adobe offers another way to consume events besides webhooks: journaling. The Adobe I/O Events Journaling API enables enterprise integrations to consume events according to their own cadence and process them in bulk. Unlike webhooks, no additional registration or other configuration is required; every enterprise integration that is registered for events is automatically enabled for journaling. Journaling data is retained for 7 days.
 
+## What is a Journal
 
+A Journal, is an ordered list of events - much like a ledger or a log where new entries (events) are added to the end of the ledger and the ledger keeps growing. Your application can start reading the ledger from any position and then continue reading "newer" entries (events) in the ledger, much like turning pages forward.
 
-Rather than webhooks, which are a _push_ model for events, journaling is a _pull_ model, in which the integration issues an API call to pull a list of events from Adobe. As with webhooks, Adobe delivers the event list as a JSON object on the following model: 
+Journaling, in contrast to webhooks, is a _pull_ model of consuming events, whereas webhooks are a _push_ model. In journaling your application will issue a series of API calls to pull batches of one or more events from the journal. The Journaling API response contains event data and the unique position in the journal for every event returned in that batch. 
 
-```json
-[
-  {
-    "events": [
-      "string"
-    ],
-    "next": "string"
-  }
-]
-```
-
-There is only one API for journaling:
+The position of an event in the journal is significant. For your application to continue reading "newer" events in the journal, the position of the last event needs to be supplied back to the Journaling API in order to fetch events "newer" than the last event.
 
-`/events/organizations/{orgId}/integrations/{intId}/{registrationId}`
+A batch of events returned by the Journaling API looks similar to the following JSON object.
 
-This API gets all events for a given event registration.
+```json
+{
+   "events": [ // an ordered list of events
+      { 
+         "position": "string", // unique position of this event in the journal
+         "event": { 
+            "key": "value" // actual event data
+         }
+      }
+   ],
+   "_page": {
+      "last": "string", // position corresponding to the last event returned in this batch
+      "count": 1 // number of events returned in this batch
+   }
+}
+```
 
-Make sure that the `I/O Management API` is added as a service in your integration (using the `Services` tab in the I/O Console), in order to be able to invoke the journaling API.
```

Comment:
I don't know why you deleted this sentence. We added it on purpose, after many people encountered problems invoking the API, since they had forgotten to add the Management API service.